src/thundercore.json

```diff
@@ -113,6 +113,17 @@
       ],
       "tags": []
     },
+    {
+      "address": "0xFCDEa9f405bc6bd85C2e90c1DEbB9F3ca7a59D8f",
+      "symbol": "DICEZ",
+      "name": "DICEZ",
```

Comment:
Token name isn't aligned with contract?

Reply:
Updated the name

Reply:
By the way, the file thundercore.json has some other issues. The last token section of RAM contains information of Oolong as well with duplicate keys.

Reply:
Thanks for the reminder, we've noticed that and fixed it.